exporter-errors/fatal-errors.json

```diff
@@ -205,6 +205,12 @@
       "categoryId": "download_imodel",
       "canUserFix": true,
       "kbLinkId": ""
+    },
+    "NoWritePermission": {
+      "description": "No 'write' permission for the selected folder.",
+      "categoryId": "upload_output_file",
+      "canUserFix": true,
+      "kbLinkId": ""

```

Comment:
If canUserFix is true, we strongly recommend writing a KB article (and adding the link here) that describes what the user should do to fix this...

Reply:
I agree. @KrBaibhav Let's add an article similar to the one with share-read permissions and update this.

Reply:
Any update @ajinkya1905 and @KrBaibhav ?